physics/cooling/include/cooling/cooler.h

```diff
@@ -0,0 +1,24 @@
+//
+// Created by Noah Kubli on 28.11.22.
+//
+
+#ifndef SPHEXA_COOLER_H
+#define SPHEXA_COOLER_H
+
+extern "C" {
+#include <grackle.h>
+}
+namespace cooling {
+
+    struct GlobalValues {
+        code_units units;
+        chemistry_data data;
+        chemistry_data_storage rates;
+    };
+
+    struct chemistry_data_
+    {
+        chemistry_data content;
+    };
```

Comment:
This is not the interface, it should be only be in `cooler.cpp`.
The declaration of `Cooler` in `cooling.hpp` is the interface. That file should be called `cooler.h`.